### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by jaspertandy** [fg2it/phantomjs-on-raspberry#6]
Hi, sorry for leaving this as an issue but I couldn't find another way to contact you. Just wanted to say thank you for compiling these for arm64/debian. You've saved me such a huge amount of headache I can't even tell you.

If you have some instructions on how you got these compiling as I tried the instructions on phantomjs' site but it fails in a docker container. I just happened to find someone had linked to this repo on a Stack Overflow!

**Comment by bonnebulle** [comment]
Am so happy to make it (finaly) works (too) but...
Some one find.serched a way to install it on Rasp4 with .deb or other way than Docker ?
A using it on a nodejs project (not using Docker)
Thanks

**Comment by jaspertandy** [comment]
@bonnebulle there's a binary in releases that I use in Debian-based Docker images:

```
wget https://github.com/fg2it/phantomjs-on-raspberry/releases/download/v2.1.1-wheezy-jessie-armv6/phantomjs
mv phantomjs /usr/local/bin/
chmod +x /usr/local/bin/phantomjs
```

Your node phantom install should look in your path for a valid binary but I have found that only Yarn does this. NPM doesn't.

**Comment by bonnebulle** [comment]
Yes nodejs module ask for a specific folder location. I finaly find the file.
Thanks for you quick answer

**Comment by bonnebulle** [comment]
Just for note @jaspertandy with your version I have this error :
`Auto configuration failed
4159390720:error:25066067:DSO support routines:DLFCN_LOAD:could not load the shared library:dso_dlfcn.c:185:filename(libssl_conf.so): libssl_conf.so: Ne peut ouvrir le fichier d'objet partagé: Aucun fichier ou dossier de ce type
4159390720:error:25070067:DSO support routines:DSO_load:could not load the shared library:dso_lib.c:244:
4159390720:error:0E07506E:configuration file routines:MODULE_LOAD_DSO:error loading dso:conf_mod.c:285:module=ssl_conf, path=ssl_conf
4159390720:error:0E076071:configuration file routines:MODULE_RUN:unknown module name:conf_mod.c:222:module=ssl_conf`
Strange
I restore an old version I build a few months ago (I dont remember how)... working without error
https://github.com/bonnebulle/phantomjs_raspberrypi4/blob/main/phantomjs

**Comment by msonowal** [comment]
@fg2it Thank you for your arm aka aarch64 builds this works on aws graviton 2 processor also
